### PR TITLE
6833 wcag 1.1.1, 2.1.1 ja 4.1.2.  suoritukset valilehdella on useita saavutettavuusongelmia

### PR DESCRIPTION
**Pull request by TeroOtavia**
Studies records list accessibility rework:
- record items have now keyboard support (space + enter will expand/shrink record info)
- record activity indicator is now focusable and opens info when focus is active
- added aria attributes where they are necessary

Resolves: #6833

### CODE REVIEW

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json [hunk 1]
```diff
@@ -490,6 +490,12 @@
       "updateError": "$t(common:notifications.updateError)",
       "updateError_matriculationEligiblity": "Loading the eligibility rights for the {{subject}} failed. Please try refreshing the page.",
       "updateError_summary": "Loading the summary information failed. Please try refreshing the page."
+    },
+    "wcag": {
+      "excersiseActivity": "Harjoitustehtäviä tehty {{percent}}%",
+      "evaluableActivity": "Arvioitavia tehtäviä tehty {{percent}}%",
```
Comment (jjlankinen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json:496]: EN locales missing in lines 495 and 496 and might be good to change the keys also to be more inline with our terminology.
```
"exercisesActivity": "Exercises done {{percent}}%",
"assignmentsActivity": "Assignments done {{percent}}%",
```

**Review by jjlankinen — CHANGES_REQUESTED** [review]

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json [hunk 2]
```diff
@@ -490,6 +490,12 @@
       "updateError": "$t(common:notifications.updateError)",
       "updateError_matriculationEligiblity": "Loading the eligibility rights for the {{subject}} failed. Please try refreshing the page.",
       "updateError_summary": "Loading the summary information failed. Please try refreshing the page."
+    },
+    "wcag": {
+      "collapseRecordInfo": "Collapse workspace record info",
+      "evaluableDonePercent": "Assignments done {{percent}}%",
```
Comment (jjlankinen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json:496]: `"evaluableDonePercent": "Assignments done {{percent}}%",`

muutetaan edelleen muotoon:

`"assignmentsActivity": "Assignments done {{percent}}%",`

Eli käytetään assignment termiä ennemin kuin keksittyä evaluable termiä (ei ole oikea sana).

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json [hunk 3]
```diff
@@ -490,6 +490,12 @@
...
+      "excersiseDonePercent": "Excersises done {{percent}}%",
```
Comment (jjlankinen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json:497]: `"excersiseDonePercent": "Excersises done {{percent}}%",`

muutetaan edelleen muotoon:

`"exercisesActivity": "Exercises done {{percent}}%",`

Eli korjataan nyt typot heti.

**Review by jjlankinen — CHANGES_REQUESTED** [review]

**Review by avukkonen — APPROVED** [review]

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json [hunk 4]
```diff
@@ -490,6 +490,12 @@
       "updateError": "$t(common:notifications.updateError)",
       "updateError_matriculationEligiblity": "Loading the eligibility rights for the {{subject}} failed. Please try refreshing the page.",
       "updateError_summary": "Loading the summary information failed. Please try refreshing the page."
+    },
+    "wcag": {
+      "collapseRecordInfo": "Collapse workspace record info",
+      "assignmentsActivity": "Assignments done {{percent}}%",
+      "exercisesActivity": "Excersises done {{percent}}%",
```
Comment (jjlankinen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json:497]: Fix the typo: `Excersises done` -> `Exercises done`

**Review by jjlankinen — CHANGES_REQUESTED** [review]

**Review by jjlankinen — APPROVED** [review]

**Pull request merged**